src/listener/block-listener-adapter.js

```diff
@@ -0,0 +1,70 @@
+/**
+* Copyright Persistent Systems 2018. All Rights Reserved.
+*
+* SPDX-License-Identifier: Apache-2.0
+*@file Implementation for block listener adapter.
+*/
+
+'use strict';
+const kafka = require('kafka-node');
+const Util = require('../comm/util');
+const logger = Util.getLogger();
```

Comment:
Add a section name for logger, like `const logger = Util.getLogger('block-listener-adapter.js')`.